Add the PresentationSize issue with AirPlay among the list of known issues (#406)
SRGSSR/pillarbox-apple: docs/KNOWN_ISSUES.md

### Workaround

Pause players playing content locally.

> ## Presentation size can't be determined when AirPlay is enable (FB12080967)

Reviewer: ```suggestion
## Presentation size cannot be determined when AirPlay is enabled (FB12080967)
```

> When AirPlay is activated, the value of the `AVPlayerItem.presentationSize` property is consistently zero.

Reviewer: ```suggestion
When AirPlay is activated the value of the `AVPlayerItem.presentationSize` property is always zero.
```

> Therefore, we encounter the challenge of discerning whether the player content corresponds to a video or audio format when AirPlay is on.

Reviewer: ```suggestion
It is therefore not possible to tell whether a video or audio is played when AirPlay is enabled.
```

Reviewer: Since both sentences are also related the newline between them should probably be dropped.
